.github/workflows/wheels.yml

```diff
@@ -114,6 +123,9 @@ jobs:
           - os: macos-latest
             python: 310
             platform_id: macosx_arm64
+          - os: macos-latest
+            python: 311
+            platform_id: macosx_arm64
 
```

Comment:
but they we would need a pre-release build for 312 on macos_arm64 somewhere, either on Azure or Cirrus.

@thomasjpfan what was the reason for this specific configuration setup for macos_arm64 again?

Reply:
I moved the latest `macos_arm64` build to Cirrus because they limited the number of credits to 50: https://cirrus-ci.org/blog/2023/07/17/limiting-free-usage-of-cirrus-ci/

macOS time is 5 times more expensive than linux: https://cirrus-ci.org/pricing/#compute-credits

With the current setup, the macOS arm builds are cross compiled on GitHub Action and the latest Python build is actually tested on CirrusCI.

Reply:
We can add this information into a comment.

Reply:
Yes, please add a comment on each side: in the Azure Pipelines config and in the Cirrus-CI config.